tests/functional/firefox/whatsnew/test_whatsnew.py

```diff
@@ -1,28 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
-import pytest
-
-from pages.firefox.whatsnew.whatsnew import FirefoxWhatsNewPage
```

Comment:
Let's please delete the associated page objects that these test files use.

Reply:
doh I thought I did... 

`/tests/pages/firefox/whatsnew/whatsnew.py`
`/tests/pages/firefox/whatsnew/whatsnew_60.py`

Are there other references I missed?

Reply:
Oh sorry, I must have blanked on those! Yes this looks good, merging!